$OpenBSD$

Disable unneeded libtgvoip, tgcalls provide everything we need; from alt linux
http://www.sisyphus.ru/cgi-bin/srpm.pl/Sisyphus/telegram-desktop/getpatch/1

Index: Telegram/CMakeLists.txt
--- Telegram/CMakeLists.txt.orig
+++ Telegram/CMakeLists.txt
@@ -31,7 +31,6 @@ get_filename_component(res_loc Resources REALPATH)
 include(cmake/telegram_options.cmake)
 include(cmake/lib_ffmpeg.cmake)
 include(cmake/lib_stripe.cmake)
-include(cmake/lib_tgvoip.cmake)
 include(cmake/lib_tgcalls.cmake)
 include(cmake/td_export.cmake)
 include(cmake/td_mtproto.cmake)
